apiserver/hack/kind-cluster-config.yaml

```diff
@@ -0,0 +1,37 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
+  kubeadmConfigPatches:
+    - |
+      kind: InitConfiguration
+      nodeRegistration:
+        kubeletExtraArgs:
+          node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 30379
```

Comment:
Are there any use cases to forward GCS? I have never heard about any users forwarding the port.

Reply:
@kevin85421 the kind configuration was lifted from the API server [official documentation] 
(https://ray-project.github.io/kuberay/components/apiserver/#swagger-support)

TL;DR here's the relevant bit
```yaml
    extraPortMappings:
      - containerPort: 30379
        hostPort: 6379
        listenAddress: "0.0.0.0"
        protocol: tcp
      - containerPort: 30265
        hostPort: 8265
        listenAddress: "0.0.0.0"
        protocol: tcp
      - containerPort: 30001
        hostPort: 10001
        listenAddress: "0.0.0.0"
        protocol: tcp
      - containerPort: 8000
        hostPort: 8000
        listenAddress: "0.0.0.0"
      - containerPort: 31888
        hostPort: 31888
        listenAddress: "0.0.0.0"
  - role: worker
  - role: worker
EOF
```

I can remove the mapping in this PR and update the documentation with a separate PR. How do you wish to proceed ?

Reply:
> I can remove the mapping in this PR and update the documentation with a separate PR.

Great!

Reply:
Fixed in https://github.com/ray-project/kuberay/pull/1301/commits/c20bb27ed42b7f1af41b2f79378fce6bcd868a3b